### ISSUE TITLE
[Bug] Compatibility with Physics mod

### ISSUE THREAD

**Issue by TqLxQuanZ** [Traben-0/Entity_Model_Features#133]
**Describe the bug**
Currently, if paired with Physics mod along with EMF, all the entity killed when the mob death animation is set to ragdoll on physics mod side, they will just disappear entirely.

**To Reproduce**
1. Go to Options -> Physics Settings -> Set to Mob Physics: Ragdoll Blood
2. Find a zombie or any entity.
3. Kill it.
4. Watch the entity disappear immediately.

**Expected behavior**
When killed the entity, ragdoll animation from Physics mod continue as normal but not from EMF overriding it to be empty.

**Screenshots**
None

**Crashlog**
No crash log.

**(please complete the following information):**
 - Minecraft Version: 1.18.2
 - EMF version 1.0.2
 - Does the issue persist with only EMF installed? Yes
 - if applicable, provide your modlist or log

**Additional context**

**Comment by Traben-0** [comment]
Yeah I beleive the compat settings for physics mod only work in 1.19+.

I'll look into it but its not a huge priority.

If you disable the compat settings in emf you should atleast restore the vanilla death fallover animation

**Comment by Ruganaskel** [comment]
I have the same problem on 1 19 2

**Comment by TqLxQuanZ** [comment]
alright thanks a lot, please do let me know if it's ported over for 1.18.2 since the immersion with both mods is incredibly great.

**Comment by Hirune30** [comment]
it's still incompatible
